src/test/java/uk/gov/companieshouse/controller/AppealControllerTest_GET.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.companieshouse.controller;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.companieshouse.service.AppealService;
+import uk.gov.companieshouse.util.TestUtil;
+
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest
+@ExtendWith(SpringExtension.class)
+public class AppealControllerTest_GET {
```

Comment:
Please add tests simulating 500 response as well.

Reply:
When retrieving data from mongoDB (getById) we shouldn't see a 500 response as the return type is Optional. However I should probably investigate how we handle database connection errors on startup

Reply:
There always can be some network issue or something so I would be surprised if repository did not return any exceptions.